### DISCUSSION TITLE
config error

### DISCUSSION THREAD

**Discussion by Zeev1988** [open-mmlab/mmaction2#2843] [Q&A]
Hi,
I'm trying to run a training script for PoseC3D model with the default config file for kintics400.
The code I try to run is:

`from mmengine import Config
from mmengine.runner import Runner

```
config_file = r'./data/custom_config.py
cfg = Config.fromfile(config_file)
cfg.load_from = r'.path/k400.pth'
runner = Runner.from_cfg(cfg)
runner.train()
```

I'm getting the following error:

> The configuration file type in the inheritance chain must match the current configuration file type, either "lazy_import" or non-"lazy_import". You got this error since you use the syntax like `with read_base(): ...` or import non-builtin module in /home/ec2-user/SageMaker/pose3d/data/../mmaction2/mmaction/configs/_base_/default_runtime.py. See more information in https://mmengine.readthedocs.io/en/latest/advanced_tutorials/config.html

I don't understand the conflicts, The config from the repo, and the code from the tutorial.
Would anyone be able to help me with this?

**Comment by Zeev1988** [comment]
had the wrong path to _base_ config in the config file

**Reply by leoheung** [reply]
Hi I met the same problem. May I know how did you solve it? Many thanks!
